Source/SpatialGDK/Public/Utils/SchemaOption.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+#include <improbable/c_worker.h>
+#include "UniquePtr.h"
+
+template <typename T>
+class TSchemaOption
```

Comment:
I would vote for putting this class in the improbable namespace and call it TOption

Reply:
Me and @improbable-valentyn had a discussion about this and we decided to leave it as is for now